src/main/java/com/Acrobot/ChestShop/Commands/AccessToggle.java

```diff
@@ -0,0 +1,40 @@
+package com.Acrobot.ChestShop.Commands;
+
+import com.Acrobot.ChestShop.Listeners.Player.PlayerInteract;
+import com.Acrobot.ChestShop.Permission;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+/**
+ * @author g--o
+ *
+ * Command for toggling chest access
+ */
+public class AccessToggle implements CommandExecutor {
+    public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
+        if (cmd.getName().equals("csaccess")) {
+            String resultMsg = "Access to shops is ";
+            Player player = (Player) sender;
+            String permNode = Permission.OTHER_NAME_ACCESS + ".*";
+
+            if (PlayerInteract.canOpenOtherShops(player)) { // @FIX: why is this deprecated?
+                Permission.setPermission(player, permNode, false);
+
+                resultMsg += ChatColor.DARK_RED + "disabled";
+            } else {
+                Permission.setPermission(player, permNode, true);
```

Comment:
Please find a better way of handling the toggling besides modifying the actual permission attachments of the player as that could lead to issues with certain permission plugins and recalculating permissions tends to be slow in general. I suggest an approach similar to the notification toggling with `/cstoggle`.

Reply:
I was asked in the issue to toggle this particular permission, it's up to @JRoy I guess.
I don't think making a list is better, considering the amount of people using this is pretty minimal.
If anything, the problem lies on the fact that your plugin doesn't use PermissionProvider interface of any sort, and does not provide a setPermission which I tried to resolve here.
What's the use of permissions if you practically encode permissions via blacklist and whitelist in the end? Pretty silly

Reply:
Not saying this is neccessarily a permenant solutionm but given the amount of users that have access to the command it'll do for now